src/utils/api.js

```diff
@@ -0,0 +1,17 @@
+export const updatePassword = async (currentPassword, newPassword, confirmPassword) => {
+    const response = await fetch('http://localhost:3000/api/v1/users/update-password', {
+      method: 'PATCH',
```

Comment:
put this url into environment variables

Reply:
blood does when you commit using amend don't show updates?
because now nothing is hardcoded my bro?